cmd/cosmos-sdk-cli/cmd/init.go

```diff
@@ -44,8 +44,12 @@ var initCmd = &cobra.Command{
 			"basecoind", shortProjectName+"d",
 			"BasecoinApp", capitalizedProjectName+"App",
 			remoteBasecoinPath, remoteProjectPath,
-			"basecoin", shortProjectName)
-		setupBasecoinWorkspace(shortProjectName, remoteProjectPath)
+			"basecoin", shortProjectName,
+			"Basecoin", capitalizedProjectName)
+		err := setupBasecoinWorkspace(shortProjectName, remoteProjectPath)
+		if err != nil {
+			return err
+		}
```

Comment:
can just use `return setupBasecoinWorkspace(shortProjectName, remoteProjectPath)`

Reply:
addressed myself - good to merge otherwise

Reply:
Sorry, was very obvious, I should have done this :) Thanks for fixing this.